docs/architecture/adr-006-non-interactive-defaults.md

```diff
@@ -210,15 +210,15 @@ func roundUpBy(cursor, v int) int {
 
 ```
 
-We can now use this function in many places, such as when we estimate the square size, calculate the number of messages used, calculate which subtree roots are needed to verify a share commitment, and calculate when to start the first message after the reserved namespaces are filled.
+We can now use this function in many places, such as when we estimate the square size, calculate the number of messages used, calculate which subtree roots are needed to verify a shared commitment, and calculate when to start the first message after the reserved namespaces are filled.
```

Comment:
[blocking] the original was fine because this commitment referenced is a "share commitment". Ref: https://github.com/celestiaorg/celestia-app/blob/1e014d594d1eedc4a0a575dba299653cad8d9247/x/blob/README.md?plain=1#L9

```suggestion
We can now use this function in many places, such as when we estimate the square size, calculate the number of messages used, calculate which subtree roots are needed to verify a shared commitment, and calculate when to start the first message after the reserved namespaces are filled.
```